torch/csrc/lazy/core/tensor_impl.cpp

```diff
@@ -148,6 +157,18 @@ void LTCTensorImpl::setup_sym_sizes() const {
     sym_sizes.push_back(sn->toSymInt());
   }
 
+  // Making the symbolic strides on LazyTensorImpl always be contiguous.
+  // Is this the right thing to do?
+  std::vector<c10::SymInt> sym_strides;
+  sym_strides.resize(rank);
+  if (rank > 0) {
+    const auto last_idx = rank - 1;
+    sym_strides.at(last_idx) = 1;
+    for (auto i = last_idx - 1; i >= 0; --i) {
+      sym_strides.at(i) = sym_strides.at(i+1) * sym_sizes.at(i+1);
+    }
+  }
```

Comment:
@Krovatkin one question I have is - how far along is LTC symbolic shapes / is it far enough along to try to get symbolic strides working with LTC in this PR?

Also, let me know if what I put seems at all reasonable to you - I basically:
-  matched LTC's `sym_sizes_` field with a `sym_strides_` field
- for simplicity (although it didn't work out that way haha), I assumed contiguous strides.
- Doing that requires `operator*` to be implemented on LTC's `torch::lazy::SymIntNodeImpl`, so I moved it around and started trying to add a few methods to it.

Right now, the tests currently fail because `torch::lazy::SymIntNodeImpl::wrap()` currently isn't implemented, and I wasn't sure if that was in a state where it's ready to work on yet.

Let me know if you think we should just do something totally different in this PR!